pancancer_evaluation/utilities/classify_utilities.py

```diff
@@ -172,7 +174,13 @@ def run_cv_cancer_type(data_model,
                        num_folds,
                        use_pancancer,
                        use_pancancer_only,
-                       shuffle_labels):
+                       shuffle_labels,
```

Comment:
I think this is where shuffle_labels is getting called and throwing the errors in checks?